Sources/Internal/Calendar+Helpers.swift

```diff
@@ -28,8 +28,7 @@ extension Calendar {
   }
 
   func firstDate(of month: Month) -> Date {
-    let firstDateComponents = DateComponents(era: month.era, year: month.year, month: month.month)
-    guard let firstDate = date(from: firstDateComponents) else {
+    guard let firstDate = date(from: month.components) else {
```

Comment:
simplified since there was already an helper to turn a `Month` into `DateComponents`